Dockerfile

```diff
@@ -2,13 +2,20 @@ FROM quay.io/centos/centos:stream8
 
 RUN dnf -y module install python39 && dnf -y install python39 python39-pip
 RUN mkdir /app
-ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugin-template-python/main/LICENSE /app
-ADD example_plugin.py /app
-ADD test_example_plugin.py /app
-ADD requirements.txt /app
+RUN chmod 777 /app
+ADD https://raw.githubusercontent.com/arcalot/arcaflow-plugin-template-python/main/LICENSE /app/
+ADD README.md /app/
+ADD poetry.lock /app/
+ADD pyproject.toml /app/
+ADD example_plugin.py /app/
+ADD test_example_plugin.py /app/
+RUN chmod +x /app/example_plugin.py /app/test_example_plugin.py
 WORKDIR /app
 
-RUN pip3 install -r requirements.txt
+RUN pip3 install poetry
+RUN poetry config virtualenvs.create false
+RUN poetry install --without dev
+RUN /app/test_example_plugin.py
```

Comment:
Does this have to run here before it is run with coverage below at line 22?

Reply:
It does not. It made it in there since where I was copying from did not have a coverage test below. This brings a good point as it may be in other plugin PR's as well.